inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java

```diff
@@ -205,20 +205,21 @@ public List<AuditVO> listByCondition(AuditRequest request) throws Exception {
                 String format = "%Y-%m-%d %H:%i:00";
                 // Support min agg at now
                 DateTimeFormatter forPattern = DateTimeFormat.forPattern("yyyy-MM-dd");
```

Comment:
`DateTimeFormatter` is thread-safe, please extract it as a static and final constant in this class, thanks.

Reply:
Done.